fe/fe-core/src/main/java/com/starrocks/server/LocalMetastore.java

```diff
@@ -2280,6 +2272,52 @@ public void replayCreateMaterializedView(String dbName, MaterializedView materia
         }
     }
 
+    public void replayAddMvPartitionNameRef(MaterializedViewPartitionNameRef ref) {
+        Database db = this.idToDb.get(ref.getDbId());
+        MaterializedView mv = ((MaterializedView) db.getTable(ref.getMvId()));
+        db.writeLock();
```

Comment:
no need to lock for replay.

Reply:
I also don't think it's needed if the journals are executed sequentially, but some replay functions added it.  e.g.
```
 public void replayAddPartition(PartitionPersistInfo info) throws DdlException {
        Database db = this.getDb(info.getDbId());
        db.writeLock();
        try {
            OlapTable olapTable = (OlapTable) db.getTable(info.getTableId());
            Partition partition = info.getPartition();

            PartitionInfo partitionInfo = olapTable.getPartitionInfo();
            if (info.isTempPartition()) {
                olapTable.addTempPartition(partition);
            } else {
                olapTable.addPartition(partition);
            }
```

Reply:
it is not necessary. remove it